deps/rabbitmq_ct_helpers/src/rabbit_ct_helpers.erl

```diff
@@ -597,9 +597,14 @@ ensure_rabbitmq_queues_cmd(Config) ->
 
 ensure_ssl_certs(Config) ->
     SrcDir = ?config(rabbitmq_ct_helpers_srcdir, Config),
+    UniqueDir = io_lib:format(
+                  "~s2-~p",
+                  [node(), erlang:unique_integer([positive,monotonic])]),
```

Comment:
I see you stopped using the name name and now use a random value. For the sake of curiosity, what did not work with the node name?

Reply:
In some suites we initialise SSL at the group level, which still caused some issues creating certificates on the same folder on the same shard. Now, we have unique cert paths for any group that needs SSL within a shard.

Reply:
Ok, thank you for clarifying!